core/basic_authorship/block_builder.hpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_BASIC_AUTHORSHIP_BLOCK_BUILDER_HPP
+#define KAGOME_CORE_BASIC_AUTHORSHIP_BLOCK_BUILDER_HPP
+
+#include "primitives/block.hpp"
+#include "primitives/extrinsic.hpp"
+
+namespace kagome::basic_authorship {
+
+  /**
+   * Creates new blocks from provided header and extrinsics
```

Comment:
Where is the header provided?  I find it slightly confusing that there's no method for it nor a hint where it should be.